utils/global-state-update-gen/src/validators.rs

```diff
@@ -55,7 +55,7 @@ pub(crate) fn generate_validators_update(matches: &ArgMatches<'_>) {
         accounts,
         transfers: vec![],
         only_listed_validators: true,
-        slash_instead_of_unbonding: true, // for consistency with the old behavior
+        slash_instead_of_unbonding: false,
```

Comment:
Not sure about changing the default here since it may brake scripts that are already using this command and expect this behaviour.
Can't we add a `--slash` parameter to the `change-validators` subcommand and leave the default to true?

Reply:
Well, I think the whole point was to make the default `false` since slashing is seen as an extreme measure, so if it does break scripts we'll need to adjust them to suit the new behaviour.  Also, this command should be treated as deprecated since the `generic` subcommand is meant to be the go-forward.

Reply:
Make sense, thanks! I also saw that we're at v0.3.0 with this tool so breaking past usage shouldn't be a concern.